beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -96,33 +68,6 @@ pub static SIGNED_BEACON_BLOCK_BELLATRIX_MAX: LazyLock<usize> =
     + types::ExecutionPayload::<MainnetEthSpec>::max_execution_payload_bellatrix_size() // adding max size of execution payload (~16gb)
     + ssz::BYTES_PER_LENGTH_OFFSET); // Adding the additional ssz offset for the `ExecutionPayload` field
 
-pub static SIGNED_BEACON_BLOCK_CAPELLA_MAX: LazyLock<usize> = LazyLock::new(|| {
```

Comment:
I think we can also remove the `FIXME` comment above

https://github.com/sigp/lighthouse/blob/2e1155490583ea654fa4ee0247fcf6556084365d/beacon_node/lighthouse_network/src/rpc/protocol.rs#L90-L93

Reply:
https://github.com/sigp/lighthouse/pull/6798/commits/4261ce0e49866782d0297667c41faf6c24faa9cc